src/main/scala/outwatch/dom/OutwatchAttributes.scala

```diff
@@ -0,0 +1,68 @@
+package outwatch.dom
+
+import outwatch.dom.helpers._
+
+/** Trait containing the contents of the `Attributes` module, so they can be
+  * mixed in to other objects if needed. This should contain "all" attributes
+  * and mix in other traits (defined above) as needed to get full coverage.
+  */
+trait OutwatchAttributes
+  extends OutWatchChildAttributes
+  with SnabbdomKeyAttributes
+  with OutWatchLifeCycleAttributes
+  with TypedInputEventProps
+  with AttributeHelpers
+
+/** OutWatch specific attributes used to asign child nodes to a VNode. */
+trait OutWatchChildAttributes {
+  /** A special attribute that takes a stream of single child nodes. */
+  lazy val child    = ChildStreamReceiverBuilder
+
+  /** A special attribute that takes a stream of lists of child nodes. */
+  lazy val children = ChildrenStreamReceiverBuilder
+}
+
+/** Outwatch component life cycle hooks. */
+trait OutWatchLifeCycleAttributes {
+  /** Lifecycle hook for component insertion. */
+  lazy val insert   = InsertHookBuilder
+
+  /** Lifecycle hook for component updates. */
+  lazy val update   = UpdateHookBuilder
+
+  /** Lifecycle hook for component destruction. */
+  lazy val destroy  = DestroyHookBuilder
+}
+
+/** Snabbdom Key Attribute */
+trait SnabbdomKeyAttributes {
+  lazy val key = KeyBuilder
+}
+
+trait TypedInputEventProps {
+  import org.scalajs.dom
+
+  /** The input event is fired when an element gets user input. */
+  lazy val onInputChecked = DomEvents.onChange.map(_.targetT[dom.html.Input].checked)
+
```

Comment:
Assuming `Input` element here is wrong, see https://github.com/OutWatch/outwatch/issues/93.